demo/assets/face/Thank You.txt

```diff
@@ -4,4 +4,4 @@ So go wild with whatever your heart
 desires!!! I hope he will be an
 amazing companion in your journey!
 
--Kana Fuyuko
\ No newline at end of file
+-Kana Fuyuko
```

Comment:
Should be reverted.

Reply:
Actually, just realized reverting this will cause file format CI to fail since it's not added as an exception, because it's not named LICENSE or NOTICE. Do we want to add it as an exception?

Reply:
Yeah, let's include it as an exception.

Reply:
Alright, done 👍